sopel_channelmgnt/channelmgnt/__init__.py

```diff
@@ -35,6 +42,9 @@ def configure(config):
     config.define_section('channelmgnt', ChannelmgntSection, validate=False)
     config.channelmgnt.configure_setting('datafile', 'Where is the datafile for channelmgnt?')
     config.channelmgnt.configure_setting('support_channel', 'What channel should users ask for help in?')
+    config.channelmgnt.configure_setting('global_chanops', 'If anybody, who should have chanop access in all channels? (if any then enter nickserv accounts)')
+    config.channelmgnt.configure_setting('log_kicks', 'Should kicks be logged to a channel? (true/false)')
```

Comment:
use channel specific config formats in datafile

Reply:
?

Reply:
log_kicks should be per channel overrideable

Reply:
I was trying to mimic sigmabot, where kicks for all channels went to the same channel

Reply:
I'm saying that's not ideal. We should allow channels to have their own logs too.